src/components/Content/Card.jsx

```diff
@@ -64,24 +64,20 @@ const Card = ({ id, name, bookmark, poster_path, title, release_date, first_air_
         <ul className="flex content-sec gap-x-3 mt-2">
         <li className="flex flex-col items-center text-white text-xs">
         <span className="mr-1">
-            { media_type === MEDIA_TYPE.MOVIES ? (
-              release_date?.slice(0,4)
-            ) : (
-              first_air_date?.slice(0,4)
-            ) }
+            { release_date?release_date.slice(0,4):first_air_date.slice(0,4)}
         </span>
         </li>
         
         <li className="flex items-center">
           <RiFilmFill className="mr-1 text-white" />
-          <span>{media_type?.toUpperCase()}</span>
```

Comment:
Why are we removing this?

Reply:
Capitalized words(media_type) was not looking nice in UI, that's why i have changed media_type only